### PR TITLE
Adds double-doors to the Nerva, firelocks to maint entrances, guardrails to the freight elevator, and a few more blob landmarks.

### PR DESCRIPTION
**Pull request by chaoko99**
…

🆑 
Adds double doors to the Nevra. 
Adds some blob landmarks.
Adds firelocks to maint entrances.
Adds guardrails to the freight elevator.
adds the requested shield diffusers to all EVA airlocks.
/ 🆑

### CODE REVIEW

File: maps/nerva/nerva-3.dmm [hunk 1]
```diff
@@ -14214,6 +14246,12 @@
 /obj/machinery/atmospherics/pipe/simple/hidden/supply,
 /turf/simulated/floor/tiled,
 /area/security/nuke_storage)
+"ZI" = (
+/obj/effect/landmark{
+	name = "blobstart"
```
Comment (Glloyd) [inline maps/nerva/nerva-3.dmm]: please no

**Comment by chaoko99** [comment]
fixed

**Comment by chaoko99** [comment]
lemme merj

File: maps/nerva/nerva-3.dmm [hunk 2]
```diff
@@ -2234,6 +2244,8 @@
 	},
 /obj/machinery/atmospherics/pipe/simple/hidden/scrubbers,
 /obj/machinery/atmospherics/pipe/simple/hidden/supply,
+/obj/machinery/door/firedoor/border_only,
```
Comment (Glloyd) [inline maps/nerva/nerva-3.dmm]: These should be regular firedoors to keep them consistent, I missed that earlier. The border ones don't do anything different anyways.

**Comment by Glloyd** [comment]
That's a quick change. I'll speedmerge it once you get that in.

**Pull request merged**